### PR TITLE
Added tests for validators

### PR DESCRIPTION
**Pull request by FelixDQ**
I also added a few misc changes
* I added pytest 
* I inverted the expected return values from the validators so that 1=valid and 0=invalid
* Added tests for the current sanity check validators

### CODE REVIEW

File: tests/validators/lib/load_fixture.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import os
```
Comment (pierresegonne) [inline tests/validators/lib/load_fixture.py:1]: why not name this file `fixture.py`?

Comment (pierresegonne) [reply tests/validators/lib/load_fixture.py:1]: or `fixtures.py`?

File: tests/validators/test_sanity_checks.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    validate_production_one_non_nan_value,
+)
+
+from .lib.load_fixture import load_fixture
+
+
+def test_validate_positive_production():
+    events = load_fixture("production_negative_values")
+    res = validate_positive_production(events)
+    assert res["2022-01-01 00:00:00+00:00"] == 1
```
Comment (pierresegonne) [inline tests/validators/test_sanity_checks.py:14]: could simplify by checking that the column is [1, 0, 1]

**Review by pierresegonne — APPROVED** [review]

File: scripts/tooling.py [hunk 3]
```diff
@@ -32,8 +32,7 @@ def lint():
 
 
 def test():
-    _run("python -u -m unittest discover tests")
-    _run("python -u -m unittest discover parsers/test")
+    _run("pytest")
```
Comment (pierresegonne) [inline scripts/tooling.py:35]: Ideally we would have added pytest in a separate PR.

Does that still cover all of the other tests?

Comment (FelixDQ) [reply scripts/tooling.py:35]: > Ideally we would have added pytest in a separate PR.

Yea, I know 🙈

>Does that still cover all of the other tests?

Yup!

File: tests/validators/test_sanity_checks.py (hunk 2 above)
Comment (FelixDQ) [reply tests/validators/test_sanity_checks.py:14]: 👍 I'll keep it in mind for other tests. I think for these I'll keep the multiple asserts as they are a bit more readable.

**Pull request merged**